ydb/docs/en/core/yql/reference/yql-core/syntax/drop_view.md

```diff
@@ -0,0 +1,27 @@
+# DROP VIEW
+
+DROP VIEW drops an existing view.
+
+## Syntax
+
+```sql
+DROP VIEW name
```

Comment:
so there's no `if exists` clause?

Reply:
Yeah. There will be an error, if the view does not exist:
`Error: Path does not exist, code: 2003`